atc/metric/emitter/newrelic.go

```diff
@@ -103,10 +104,21 @@ func (emitter *NewRelicEmitter) emitPayload(logger lager.Logger, payload fullPay
 	resp, err := emitter.client.Do(req)
 
 	if err != nil {
-		logger.Error("failed-to-send-request",
+		logger.Error("new-relic-failed-to-send-request",
 			errors.Wrap(metric.ErrFailedToEmit, err.Error()))
 		return
 	}
+	if resp.StatusCode < 200 || resp.StatusCode > 299 {
+		bodyBytes, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			logger.Info("new-relic-failed-to-send-request",
```

Comment:
Wdyt of having this error logged with a different action from the next `logger.Info`?

I think this `failed-to-send-request` is more of a `failed-to-read-response-body` 🤔  

wdyt?